lib/src/replay/mask/posthog_mask_controller.dart

```diff
@@ -32,15 +31,41 @@ class PostHogMaskController {
       PostHogMaskController._privateConstructor(
           Posthog().config?.sessionReplayConfig);
 
-  Future<List<Rect>?> getCurrentScreenRects() async {
+  /// Extracts a flattened list of [ElementData] objects representing the
+  /// renderable elements in the widget tree.
+  ///
+  /// This method traverses the tree of [ElementData] objects and returns a
+  /// list of elements that have no children or only one child.
+  ///
+  /// The method is designed to extract the elements that are directly
+  /// renderable on the screen.
+  ///
+  /// **Returns:**
+  ///   - `List<ElementData>`: A list of [ElementData] objects representing the
+  ///     renderable elements.
+  ///
+  List<ElementData>? getCurrentWidgetsElements() {
```

Comment:
```suggestion
  List<ElementData>? _getCurrentWidgetsElements() {
```

can be private I assume

Reply:
we call it here

https://github.com/PostHog/posthog-flutter/blob/25a3690d6fe4cb5d53516c7b70161fffaabd106e/lib/src/replay/screenshot/screenshot_capturer.dart#L103-L106